xml/System.Reflection/Assembly.xml

```diff
@@ -4128,7 +4128,17 @@ Note: In <see href="https://go.microsoft.com/fwlink/?LinkID=247912">.NET for Win
         <param name="assemblyString">The long or short form of the assembly name.</param>
         <summary>Loads an assembly with the specified name.</summary>
         <returns>The loaded assembly.</returns>
-        <remarks>
+        <remarks FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;net-5.0;net-6.0;net-7.0;net-8.0;net-9.0">
```

Comment:
Remarks can't be versioned like this unfortunately. You just have to annotate them in the text itself where they are version specific.